website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -36,7 +36,7 @@ Consul supports **four major Envoy releases** at the beginning of each major Con
 
 | Consul Version      | Compatible Envoy Versions                                                          |
 | ------------------- | -----------------------------------------------------------------------------------|
-| 1.12.x              | 1.22.0, 1.21.1, 1.20.2, 1.19.3                                                     |
+| 1.12.x              | 1.22.1, 1.21.3, 1.20.4, 1.19.5                                                     |
 | 1.11.x              | 1.20.2, 1.19.3, 1.18.6, 1.17.4<sup>1</sup>                                         |
```

Comment:
Oh, good call. I usually only consider these upgrade PRs for the latest release series, but this is different since it's patches. I will create a separate PR for 1.11.x.

Reply:
For posterity, this is the PR for 1.11: https://github.com/hashicorp/consul/pull/13434